lib/opal/hike_path_finder.rb

```diff
@@ -7,12 +7,61 @@ def initialize(paths = Opal.paths)
       super()
       append_paths(*paths)
       append_extensions '.js', '.js.rb', '.rb', '.opalerb'
+      @patterns = {}
     end
 
-    def find path
+    def find path, options={}
```

Comment:
Originally, path_reader's `expand` method was using this with an additional option but ended up having to write a new method anyways. This could be removed.